src/test/codegen/remap_path_prefix/issue-73167-remap-std.rs

```diff
@@ -0,0 +1,15 @@
+// ignore-windows
+
+// compile-flags: -g  -C no-prepopulate-passes -Z remap-rust-src-base=/rustc
```

Comment:
Do you need to add a pseudo-SHA value here in order for it to conform to the same regex as regular remapped paths? E.g. `-Z remap-rust-src-base=/rustc/xyz`.

Reply:
Added that. I don't have time to run the tests locally atm so let's see if the CI works